docs/pages/grafana.mdx

```diff
@@ -0,0 +1,217 @@
+
+That's it! Now you can scroll to the bottom of the configuration page and click **Save & Test**. There should be green alert with a title **Successfully queried the Prometheus API.**
+
+### Unauthenticated Prometheus
+
+If for any reason it's not possible to attach authorization token for Grafana, Pelican also allows you to turn off Prometheus authentication. You can toggle the configuration value `Monitoring.PromQLAuthorization` and set it to `false` to allow unauthenticated access to Prometheus. You can either set the value by the configuration file or by the environment variable.
+
+#### Set through configuration file
+
+Pelican configuration file is located at `/etc/pelican/pelican.yaml` for root users, or `~/.config/pelican/pelican.yaml` for non-root users
+
+Append the following lines to your `pelican.yaml` file:
+
+```yaml filename="pelican,yaml"
+# Other configurations ...
+
+Monitoring:
+    PromQLAuthorization: false
+```
+
+> Note that you need to restart the Pelican server to apply the changes.
+
+#### Set through environment variable
+
+The environment variable for the configuration is `PELICAN_MONITORING_PROMQLAUTHORIZATION`.
+
+You can either `export` this environment variable or append it before your command to run Pelican.
+
+```bash
+export PELICAN_MONITORING_PROMQLAUTHORIZATION=false
+```
+
+or
+
+```bash
+PELICAN_MONITORING_PROMQLAUTHORIZATION=false pelican origin serve
+```
+
+Once you configured the authentication either in Grafana or in Pelican, click **Save & Test** to see if the connection can be established.
+
+# Build Monitroing Dashbord
+
+With Grafana connected to Pelican Prometheus instance, you are able to build visualization with the metrics Pelican collects. This section describes how to get started with a template dashboard configuration for a _Pelican origin server_. For a detailed guide on how to create a Grafana dashboard, refer to [Grafana documentation](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/create-dashboard/).
+
+## Before start
+
+1. Make sure your Pelican origin server is running
+
+2. Download the template dashboard file from [here](https://github.com/PelicanPlatform/pelican/blob/main/docs/public/origin-dashboard-template.json)
+
+
+## Modify dashboard template
+
+There are a couple of placeholders need to be changed before you can import the dashboard template to Gafana. To do a batch replace, it is recommended that you use a text editor to find and replace placeholders to corresponding values.
+
+* Replace `<YOUR-DATA-SOURCE-UID>` to the id of your Prometheus data source. You can find the ID on the data source configuration page.
+
+    1. Navigate to the configuration page
+    2. Copy the ID from your browser's URL box, which should have a value similar to
+
+        ```console
+        http://localhost:3000/connections/datasources/edit/fc2115e0-ede3-4fdb-9056-ecefa1b3160a
+        ```
```

Comment:
Unsure if this is just in my view, but it looks like (at least in Github) that the ``` didn't close the console block here. But again, that could just be GitHub. Everything below it is still the blue of the console block, oddly enough.

Reply:
That should be fine, my local markdown preview works